Integrations don't start on master

## This is a...

<pre><code>
[ ] Feature request
[x] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Documentation issue or request
</code></pre>

## Description
After publishing an integration, the integration builds successfully but the deploy pod finishes in a few seconds and the integration pod gets created but errors out after just booting up.

Attaching log from the integration pod:
```
Starting the Java application using /opt/run-java/run-java.sh ...
exec java -javaagent:/opt/jolokia/jolokia.jar=config=/opt/jolokia/etc/jolokia.properties -javaagent:/opt/prometheus/jmx_prometheus_javaagent.jar=9779:/tmp/src/prometheus-config.yml -XX:+UseParallelGC -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:MinHeapFreeRatio=20 -XX:MaxHeapFreeRatio=40 -XX:+ExitOnOutOfMemoryError -cp . -jar /deployments/project-0.1-SNAPSHOT.jar
OpenJDK 64-Bit Server VM warning: If the number of processors is expected to increase from one, then you should configure the number of parallel GC threads appropriately using -XX:ParallelGCThreads=N
I> No access restrictor found, access to any MBean is allowed
Jolokia: Agent started with URL https://172.17.0.11:8778/jolokia/ 
      _______.                 _               _
    /       |                 | |             (_)
   |   (----`_   _  ____    _ | |  ____   ___  _   ___
    \   \   | | | ||  _ \  / || | / _  ) /___)| | /___)
.----)   |  | |_| || | | |( (_| |( (/ / |___ || ||___ |
|_______/    \__  ||_| |_| \____| \____)(___/ |_|(___/
============ (____/ ===================================
:: Integration ::  :: v
2019-04-17 07:00:02.440  INFO 1 --- [           main] io.syndesis.example.Application          : Starting Application on i-test-4-9prwh with PID 1 (/deployments/project-0.1-SNAPSHOT.jar started by jboss in /deployments)
2019-04-17 07:00:02.455 DEBUG 1 --- [           main] io.syndesis.example.Application          : Running with Spring Boot v1.5.16.RELEASE, Spring v4.3.19.RELEASE
2019-04-17 07:00:02.456  INFO 1 --- [           main] io.syndesis.example.Application          : No active profile set, falling back to default profiles: default
2019-04-17 07:00:02.667  INFO 1 --- [           main] ationConfigEmbeddedWebApplicationContext : Refreshing org.springframework.boot.context.embedded.AnnotationConfigEmbeddedWebApplicationContext@3ec300f1: startup date [Wed Apr 17 07:00:02 UTC 2019]; root of context hierarchy
2019-04-17 07:00:03.453  WARN 1 --- [           main] ationConfigEmbeddedWebApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanDefinitionStoreException: Failed to process import candidates for configuration class [io.syndesis.example.Application]; nested exception is java.lang.IllegalStateException: Unable to read meta-data for class  io.syndesis.integration.runtime.sb.jmx.IntegrationMetadataAutoConfiguration
2019-04-17 07:00:03.613 ERROR 1 --- [           main] o.s.boot.SpringApplication               : Application startup failed
org.springframework.beans.factory.BeanDefinitionStoreException: Failed to process import candidates for configuration class [io.syndesis.example.Application]; nested exception is java.lang.IllegalStateException: Unable to read meta-data for class  io.syndesis.integration.runtime.sb.jmx.IntegrationMetadataAutoConfiguration
	at org.springframework.context.annotation.ConfigurationClassParser.processDeferredImportSelectors(ConfigurationClassParser.java:561) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.annotation.ConfigurationClassParser.parse(ConfigurationClassParser.java:187) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.annotation.ConfigurationClassPostProcessor.processConfigBeanDefinitions(ConfigurationClassPostProcessor.java:308) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.annotation.ConfigurationClassPostProcessor.postProcessBeanDefinitionRegistry(ConfigurationClassPostProcessor.java:228) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanDefinitionRegistryPostProcessors(PostProcessorRegistrationDelegate.java:272) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(PostProcessorRegistrationDelegate.java:92) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.invokeBeanFactoryPostProcessors(AbstractApplicationContext.java:687) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:525) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:122) ~[spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:693) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:360) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:303) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1118) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1107) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at io.syndesis.example.Application.main(Application.java:13) [classes!/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_191]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_191]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_191]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_191]
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48) [project-0.1-SNAPSHOT.jar:na]
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87) [project-0.1-SNAPSHOT.jar:na]
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50) [project-0.1-SNAPSHOT.jar:na]
	at org.springframework.boot.loader.PropertiesLauncher.main(PropertiesLauncher.java:595) [project-0.1-SNAPSHOT.jar:na]
Caused by: java.lang.IllegalStateException: Unable to read meta-data for class  io.syndesis.integration.runtime.sb.jmx.IntegrationMetadataAutoConfiguration
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter$AutoConfigurationClass.getAnnotationMetadata(AutoConfigurationSorter.java:217) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter$AutoConfigurationClass.getAnnotationValue(AutoConfigurationSorter.java:198) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter$AutoConfigurationClass.readBefore(AutoConfigurationSorter.java:186) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter$AutoConfigurationClass.<init>(AutoConfigurationSorter.java:158) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter$AutoConfigurationClasses.<init>(AutoConfigurationSorter.java:115) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter.getInPriorityOrder(AutoConfigurationSorter.java:57) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationImportSelector.sort(AutoConfigurationImportSelector.java:241) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationImportSelector.selectImports(AutoConfigurationImportSelector.java:98) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.context.annotation.ConfigurationClassParser.processDeferredImportSelectors(ConfigurationClassParser.java:552) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	... 22 common frames omitted
Caused by: java.io.FileNotFoundException: class path resource [ io/syndesis/integration/runtime/sb/jmx/IntegrationMetadataAutoConfiguration.class] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:172) ~[spring-core-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.core.type.classreading.SimpleMetadataReader.<init>(SimpleMetadataReader.java:50) ~[spring-core-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.core.type.classreading.SimpleMetadataReaderFactory.getMetadataReader(SimpleMetadataReaderFactory.java:102) ~[spring-core-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.boot.type.classreading.ConcurrentReferenceCachingMetadataReaderFactory.createMetadataReader(ConcurrentReferenceCachingMetadataReaderFactory.java:89) ~[spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.type.classreading.ConcurrentReferenceCachingMetadataReaderFactory.getMetadataReader(ConcurrentReferenceCachingMetadataReaderFactory.java:76) ~[spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.core.type.classreading.SimpleMetadataReaderFactory.getMetadataReader(SimpleMetadataReaderFactory.java:80) ~[spring-core-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.boot.autoconfigure.AutoConfigurationSorter$AutoConfigurationClass.getAnnotationMetadata(AutoConfigurationSorter.java:213) ~[spring-boot-autoconfigure-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	... 30 common frames omitted

```

--- a/app/integration/runtime-springboot/pom.xml
+++ b/app/integration/runtime-springboot/pom.xml
@@ -138,6 +138,11 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.camel</groupId>
+      <artifactId>camel-servlet-starter</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
       <groupId>io.atlasmap</groupId>
       <artifactId>camel-atlasmap</artifactId>
       <scope>test</scope>

--- a/app/integration/runtime-springboot/src/main/resources/META-INF/spring.factories
+++ b/app/integration/runtime-springboot/src/main/resources/META-INF/spring.factories
@@ -17,7 +17,7 @@
 
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
     io.syndesis.integration.runtime.sb.IntegrationRuntimeAutoConfiguration,\
-    io.syndesis.integration.runtime.sb.SyndesisHttpConfiguration, \
+    io.syndesis.integration.runtime.sb.SyndesisHttpConfiguration,\
     io.syndesis.integration.runtime.sb.jmx.IntegrationMetadataAutoConfiguration,\
     io.syndesis.integration.runtime.sb.logging.IntegrationLoggingAutoConfiguration
 

--- a/app/integration/runtime-springboot/src/test/java/io/syndesis/integration/runtime/sb/ApplicationTest.java
+++ b/app/integration/runtime-springboot/src/test/java/io/syndesis/integration/runtime/sb/ApplicationTest.java
@@ -1,0 +1,52 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.integration.runtime.sb;
+
+
+import io.syndesis.integration.runtime.sb.jmx.IntegrationMetadataAutoConfiguration;
+import io.syndesis.integration.runtime.sb.logging.IntegrationLoggingAutoConfiguration;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DirtiesContext
+@RunWith(SpringRunner.class)
+@SpringBootApplication
+@SpringBootTest(
+    properties = {
+        "spring.main.banner-mode = off",
+        "logging.level.io.syndesis.integration.runtime = DEBUG"
+    }
+)
+public class ApplicationTest {
+    @Autowired
+    private ApplicationContext applicationContext;
+
+    @Test
+    public void testApplicationSetup() {
+        assertThat(applicationContext.getBeansOfType(IntegrationRuntimeAutoConfiguration.class)).hasSize(1);
+        assertThat(applicationContext.getBeansOfType(IntegrationMetadataAutoConfiguration.class)).hasSize(1);
+        assertThat(applicationContext.getBeansOfType(IntegrationLoggingAutoConfiguration.class)).hasSize(1);
+        assertThat(applicationContext.getBeansOfType(SyndesisHttpConfiguration.class)).hasSize(1);
+    }
+}

